src/internal/resize-handle/styles.scss

```diff
@@ -7,16 +7,13 @@
 
 .handle {
   cursor: nwse-resize;
+  @include shared.handle();
 
   @include shared.with-direction("rtl") {
     cursor: nesw-resize;
   }
 }
 
-.handle:not(.active):focus-visible {
-  @include shared.focus-highlight();
-}
-
```

Comment:
The focus style is handled by the `InternalDragHandle` from the components package.